api/v1alpha1/edgeworkload_webhook.go

```diff
@@ -101,6 +101,12 @@ func (r *EdgeWorkload) validate() error {
 		} else {
 			containersNames[container.Name] = struct{}{}
 		}
+
+		for _, port := range container.Ports {
+			if port.HostPort == 9100 {
+				notValidPaths = append(notValidPaths, containersMsg(container, "ports"))
```

Comment:
nit: to be consistent with the other if blocks, I think `ports` may be replaced with `port`

Reply:
I agree with the need to be consistent, but WDYT about `ports.hostPort`? to be consistent with lines 66-85